templates/default/server.xml.erb

```diff
@@ -52,6 +52,8 @@
 
         <Engine name="Catalina" defaultHost="localhost">
 
+            <Alias><%= node['fqdn'] %></Alias>
```

Comment:
I would use a new attr which defaults to `node['fqdn']`. A lot of times a service name is used instead of the host FQDN.